src/r5py/r5/transport_network.py

```diff
@@ -62,6 +65,60 @@ def __init__(self, osm_pbf, gtfs=[], build_config={}):
         )
         self._transport_network.transitLayer.buildDistanceTables(None)
 
+    @classmethod
+    def from_directory(cls, path):
+        """
+        Find input data in `path`, load an `r5py.TransportNetwork`.
+
+        This mimicks r5r’s behaviour to accept a directory path
+        as the only input to `setup_r5()`.
+
+        If more than one OpenStreetMap extract (`.osm.pbf`) is
+        found in `path`, the (alphabetically) first is used.
+        In case *no* OpenStreetMap extract is found, a `FileNotFound`
+        exception is raised. Any and all GTFS data files are used.
+
+        Arguments:
+        ----------
+        path : str
+            directory path in which to search for GTFS and .osm.pbf files
+
+        Returns:
+        --------
+        TransportNetwork
+            A fully initialised r5py.TransportNetwork
+        """
+        path = pathlib.Path(path)
+        try:
+            potential_osm_pbf_files = sorted(path.glob("*.osm.pbf"))
+            osm_pbf = potential_osm_pbf_files[0]
+            if len(potential_osm_pbf_files > 1):
+                warnings.warn(
+                    (
+                        "Found more than one OpenStreetMap extract file (`.osm.pbf`), "
+                        + "using alphabetically first one ({})"
+                    ).format(osm_pbf.name),
+                    warnings.RuntimeWarning,
```

Comment:
Minor comment, but I would recommend using f-strings instead of `.format()` because the readability of f-strings is much better. I.e. this would become:
```python
f"Found more than one OpenStreetMap extract file (`.osm.pbf`), "
f"using alphabetically first one {osm_pbf.name}."
```
Also notice that you don't need to add the `+` character here to combine multiple text lines.

Reply:
Yes, these are only str, so yes, I can change them to f-strings